src/react/components/pages/predict/predictPage.tsx

```diff
@@ -196,7 +203,7 @@ export default class PredictPage extends React.Component<IPredictPageProps, IPre
                             {!this.state.predictionLoaded &&
                                 <div className="loading-container">
                                     <Spinner
-                                        role="alert"
```

Comment:
I think you just need to set the ariaLive attribute (Politeness setting for label update announcement) of the spinner to assertive. By default it is polite. https://developer.microsoft.com/en-us/fabric#/controls/web/spinner

Reply:
I tried, it does not work

Reply:
With both ariaLabel and ariaLive?